src/util/sample.cpp

```diff
@@ -108,6 +108,10 @@ void SampleUtil::applyGain(CSAMPLE* pBuffer, CSAMPLE_GAIN gain,
 // static
 void SampleUtil::applyRampingGain(CSAMPLE* pBuffer, CSAMPLE_GAIN old_gain,
         CSAMPLE_GAIN new_gain, SINT numSamples) {
+    if (old_gain == new_gain) {
```

Comment:
This condition is also checked below. I think the bypass below could be removed.